recipe/build-lib.sh

```diff
@@ -1,5 +1,25 @@
 CUDA_CONFIG_ARG=""
 if [ ${cuda_compiler_version} != "None" ]; then
+    # for documentation see e.g.
+    # docs.nvidia.com/cuda/cuda-c-best-practices-guide/index.html#building-for-maximum-compatibility
+    # docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html#major-components__table-cuda-toolkit-driver-versions
+    # docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/index.html#gpu-feature-list
+
+    # the following are all the x86-relevant gpu arches; for building aarch64-packages, add: 53, 62, 72
```

Comment:
Should we go ahead and add a `if` for this case in preparation?

Reply:
I propose to cross this bridge when we come to it. ;-)
My comment *is* the preparation :yum: